#346: added changelog.md file, renamed dist file (#347)
exasol/virtual-schemas: doc/changes/changes-4.0.1.md

# Virtual Schemas 4.0.1, released 24.06.2020

## Summary 

> In this release we introduced the security risks check with `ossindex-maven-plugin`. We removed found security risks in the third-party transient test dependencies.

Reviewer: ```suggestion
In this release we introduced the security risks check with `ossindex-maven-plugin`. We removed the reported security risks by updating or excluding third-party transient test dependencies. Those were used in the integration tests of the Virtual Schema for Hive.
```

> We also adjusted the naming for the attached JAR file:
> Old naming pattern: virtualschema-jdbc-adapter-dist-<dialects version>.jar
> New naming pattern: virtual-schema-dist-<virtual schema common jdbc version>-bundle-<dialects version>.jar

Reviewer: ```suggestion
We also adjusted the naming for the attached JAR file:

Old naming pattern: `virtualschema-jdbc-adapter-dist-<dialects version>.jar`

New naming pattern: `virtual-schema-dist-<virtual schema common jdbc version>-bundle-<dialects version>.jar`
```

The new naming pattern helps users to distinguish minor dialect changes from major adapter changes affecting all dialects.

## Bug Fixes
 
* #336: Fixed broken links in the documentation.
* #343: Added ossindex-maven-plugin plugin and removed found security risks in transient dependencies.

## Features / Enhancements
 

> * #340:  Added MySQL dialect integration test.

Reviewer: ```suggestion
* #340: Added MySQL dialect integration test.
```
